qualtran/bloqs/chemistry/sparse/prepare.py

```diff
@@ -102,12 +102,12 @@ def _add(p_indx: int, q_indx: int, r_indx: int, s_indx: int):
         _add(q, q, q, p)
     for p in range(num_spat):
         _add(p, p, p, p)
-    eris_eight = np.array(eris_eight)
-    pqrs_indx = np.array(pqrs_indx)
-    keep_indx = np.where(np.abs(eris_eight) > drop_element_thresh)
-    eris_eight = eris_eight[keep_indx]
-    pqrs_indx = pqrs_indx[keep_indx[0]]
-    return np.concatenate((tpq_indx, pqrs_indx)), np.concatenate((tpq_sparse, eris_eight))
+    eris_eight_np = np.array(eris_eight)
+    pqrs_indx_np = np.array(pqrs_indx)
+    keep_indx = np.where(np.abs(eris_eight_np) > drop_element_thresh)
+    eris_eight_np = eris_eight_np[keep_indx]
+    pqrs_indx_np = pqrs_indx_np[keep_indx[0]]
```

Comment:
this sort of thing shouldn't be necessary with mypy `allow_redefinition = true` which should already be in the mypy.ini file; which I actually feel quite strongly about :)

Reply:
Hmmm... it was giving me an error and I was using the mypy.ini which has` "allow_redefinition = true"` in it.  I guess I could investigate further.

Reply:
haha I appreciate it